ModelBundle/Validator/Constraints/UniqueFieldIdContentTypeValidator.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace OpenOrchestra\ModelBundle\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+
+/**
+ * Class UniqueFieldIdContentTypeValidator
+ */
+class UniqueFieldIdContentTypeValidator extends ConstraintValidator
+{
+
+    /**
+     * @param mixed $value
+     * @param Constraint $constraint
+     */
+    public function validate($value, Constraint $constraint)
+    {
+        $fields = $value->getFields();
+        $fieldsId = array();
+
+        foreach ($fields as $field) {
+            $fieldId = $field->getFieldId();
+            if (in_array($fieldId, $fieldsId)){
+                $this->context->buildViolation($constraint->message)
```

Comment:
Where will you set your violation ?
As a global error, or as an error just for the second fieldId ?